runtime/src/iree/schemas/cpu_feature_bits.inl

```diff
@@ -114,3 +114,11 @@ IREE_CPU_FEATURE_BIT(X86_64, 0, 32, AVX512FP16, "avx512fp16")
 IREE_CPU_FEATURE_BIT(X86_64, 0, 50, AMXTILE, "amx-tile")
 IREE_CPU_FEATURE_BIT(X86_64, 0, 51, AMXINT8, "amx-int8")
 IREE_CPU_FEATURE_BIT(X86_64, 0, 52, AMXBF16, "amx-bf16")
+
+//===----------------------------------------------------------------------===//
+// IREE_ARCH_RISCV_64 / riscv64
+//===----------------------------------------------------------------------===//
+
+// General features and high-level switches.
+// RISCV vector extension. bit: 'V' - 'A' = 21
+IREE_CPU_FEATURE_BIT(RISCV_64, 0, 21, RVV, "rvv")
```

Comment:
The bit used here does not have to match the bit used in hwcap.  This bit is going to be used on all OSes, while hwcap is Linux-specific. So just replace 21 here by 0.

Reply:
I mean, unless you really think that this bit-naming scheme is universally useful (because, maybe, it's idiomatic that 'v' is used to refer to 'vector extension' ?)   Basically - feel free to use the bit you want, just don't think you're beholden to the bit layout of Linux hwcap.

Reply:
Ok, I think not all extensions will be needed in cpu_features_bits. Seem it starts from 0 is clearer.